catalogData/redis32/ha/3x/k8s/sentinel-proxy-service.json

```diff
@@ -0,0 +1,23 @@
+{
+  "apiVersion": "v1",
+  "kind": "Service",
+  "metadata": {
+    "labels": {
+      "app": "redis-sentinel-proxy"
+    },
+    "name": "redis-sentinel-proxy"
+  },
+  "spec": {
+    "ports": [
+      {
+        "name": "redis-sentinel-proxy",
+        "port": 6379,
+        "protocol": "TCP"
+      }
+    ],
+    "selector": {
```

Comment:
Most of this has been refactored, but I still need to run through a bit of this.